incubator/model-core/src/main/java/io/aklivity/zilla/runtime/model/core/internal/config/RangeConfigAdapter.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.model.core.internal.config;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class RangeConfigAdapter
```

Comment:
The config adapter design pattern instantiates the adapter, and then calls `adaptToJson` and `adaptFromJson` as needed.

Let's follow the same pattern here, using `adaptToString` and `adaptFromString` instead.